flang/test/Driver/optimization-remark.f90

```diff
@@ -19,7 +19,7 @@
 ! RUN: %flang %s -O2 -Rpass=loop -S %{output} 2>&1 | FileCheck %s --check-prefix=PASS-REGEX-LOOP-ONLY
 
 ! Check valid -Rpass-missed regex
-! RUN: %flang %s -O2 -Rpass-missed=loop -S %{output} 2>&1 | FileCheck %s --check-prefix=MISSED-REGEX-LOOP-ONLY
+! RUN: %flang %s -O2 -Rpass-missed=loop -S %{output} 2>&1 | FileCheck %s --allow-empty --check-prefix=MISSED-REGEX-LOOP-ONLY
```

Comment:
I can see there are other run lines which also contain `--allow-empty`, but I'm worried that these could prevent us from noticing missing optimisation remarks on platforms which do support them. I think it would be better to mark this test as unsupported on riscv.